va_explorer/va_data_management/models.py

```diff
@@ -584,6 +584,15 @@ class VerbalAutopsy(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
 
+    # function to tell if VA had any coding errors
+    def any_errors(self):
+        return len([e for e in self.coding_issues.all() if e.severity=='error']) > 0
```

Comment:
I think you could simplify this slightly by using [exists](https://docs.djangoproject.com/en/3.1/ref/models/querysets/#django.db.models.query.QuerySet.exists) on the queryset:

`return self.coding_issues.filter(severity='error').exists()`

That way you don't need to loop through all the results.

Reply:
good find! Hadn't seen the exists() command before. Will make that update